### ISSUE TITLE
DKIM lost with SRS

### ISSUE THREAD

**Issue by wibimaster** [docker-mailserver/docker-mailserver#2526]
# Subject

I would like some feedback concerning a use case

## Description

Hi !

I'm using your project to build a mailserver on the `aaaaaa.com` domain, and everything works fine (SPF, DKIM, DMARC passed).

On my server I have another domain, `bbbbbb.com`.

I would like to redirect all emails for `contact@bbbbbb.com` to another external mail `toto@gmail.com`, so I add the alias :

`./setup.sh alias add contact@bbbbbb.com toto@gmail.com`

And I enabled SRS to avoid spam filters.

Now I get the forwarded emails, with no warning on gmail, and with the label "Sent via aaaaaa.com" (when Gmail detect the SRS).

But I wonder why in the mail sources DMARC fails and I can't find any DKIM signature ; does the SRS process remove it ?
Is there a way to use SRS but keep the DKIM and a valid DMARC ?

Thanks !

**Comment by casperklein** [comment]
> does the SRS process remove it ?

I don't think so. Are you sure, the email is signed in the first place by the original sender? SRS should only override the envelop header.
[This](https://serverfault.com/questions/919376/dmarc-fails-on-forwarded-mails-without-dkim) could be related.
